client/src/components/reader/index.js

```diff
@@ -6,6 +6,8 @@ export AppearanceMenuButton from './AppearanceMenuButton';
 export Footer from './Footer';
 export FooterMenu from './FooterMenu';
 export Header from './Header';
+export ReturnMenuButton from './ReturnMenuButton';
```

Comment:
My preference for something like this would be this sort of structure:

```
Reader/ReturnMenu/index.js
Reader/ReturnMenu/Button.js
Reader/ReturnMenu/Body.js
```
Reader would export ReturnMenu, and in JSX you would access them with ReturnMenu.Button and ReturnMenu.Body